README.md

```diff
@@ -1,13 +1,14 @@
-# PlayHT API SDK
+# Play API SDK
 
-**pyht** is a Python SDK for [PlayHT's AI Text-to-Speech API](https://play.ht/text-to-speech-api/). PlayHT builds conversational voice AI models for realtime use cases. With **pyht**, you can easily convert text into high-quality audio streams with humanlike voices.
+**pyht** is a Python SDK for [Play's AI Text-to-Speech API](https://play.ht/text-to-speech-api/). Play builds conversational voice AI models for realtime use cases. With **pyht**, you can easily convert text into high-quality audio streams with humanlike voices.
 
-Currently the library supports only streaming text-to-speech. For the full set of functionalities provided by the PlayHT API such as [Voice Cloning](https://docs.play.ht/reference/api-create-instant-voice-clone), see the [PlayHT docs](https://docs.play.ht/reference/api-getting-started)
+Currently the library supports only streaming and non-streaming text-to-speech. For the full set of functionalities provided by the Play API such as [Voice Cloning](https://docs.play.ht/reference/api-create-instant-voice-clone), see the [Play docs](https://docs.play.ht/reference/api-getting-started)
 
 ## Features
 
 - Stream text-to-speech in real-time, synchronous or asynchronous.
```

Comment:
NIT: Reads better given the next line imo.
```suggestion
- Generate Stream text-to-speech in real-time, synchronous or asynchronous.
```

Reply:
I see what you're saying but don't agree, "stream" is the generally used verb for this use-case